packages/bpk-component-calendar/examples.js

```diff
@@ -77,13 +73,12 @@ const CalendarGridExample = () => (
     formatMonth={formatMonth}
     formatDateFull={formatDateFull}
     DateComponent={BpkCalendarDate}
-    showWeekendSeparator
     preventKeyboardFocus
   />
 );
 
 const CalendarGridAndHeaderExample = () => (
-  <div>
+  <div style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
Why do we have different styles for header and grid in terms of width? are they not generally supposed to be used together?
Also, might need a similar fix for the Week example

Reply:
This is due to when they are being used independently for custom part of the calendar - so when people are making custom calendars instead of using just `BpkCalendar` so building up individual parts they then need to take these changes into account. So this is an example of when using the calendar in separate parts

Reply:
>Also, might need a similar fix for the Week example

Good spot just seen that!